portal/views/crossdomain.py

```diff
@@ -12,7 +12,11 @@
 def crossdomain(
         origin=None,
         methods=None,
-        headers=('Authorization', 'X-Requested-With', 'X-CSRFToken'),
+        headers=(
+            'Authorization',
+            'X-Requested-With',
+            'X-CSRFToken',
+            'Content-Type'),
```

Comment:
I think you already added `Content-Type` to every place you're calling the `@crossdomain` decorator.

If I understand correctly and it's duplicated, I would remove it from the default `headers` here, or remove it from each place you added the `@crossdomain`